maintenance_evolution.es.Rmd

```diff
@@ -207,6 +207,21 @@ No es necesario que cambies los test de las funciones obsoletas hasta que caduqu
 
 Una vez que se caduca una función, sus tests se eliminan sin más.
 
+## Renombrar paquetes {#renaming-packages}
+
+Renombrar un paquete que está en desarrollo temprano está bien.
+Puede ser la oportunidad, antes de la revisión, de cumplir con nuestros [consejos de nomenclatura](#naming-your-package).
+
+Renombrar un paquete que ya está ampliamente adoptado y/o publicado en CRAN es problemático.
+CRAN tiene una [política](https://cran.r-project.org/web/packages/policies.html) que establece que los nombres de paquetes en CRAN son persistentes y en general no está permitido cambiar el nombre de un paquete.
+El paquete con su antiguo nombre puede ser una dependencia de paquetes, scripts, y aparecer en documentación y entradas de blog, etc.
```

Comment:
```suggestion
El paquete con su antiguo nombre puede ser una dependencia en otros paquetes, scripts, y aparecer en publicaciones científicas, documentación y artículos de blog, etc.
```

Reply:
I already added "scientific publications" and "documentation" to the Spanish text

Reply:
thank you!! I was about to do so